CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Changelog
 
+### Web3
+
+- Added new design for NFT spam management in Brave Wallet. ([#8205](https://github.com/brave/brave-ios/issues/8205))
+- Added bottom tab bar for Brave Wallet. ([#8166](https://github.com/brave/brave-ios/issues/8166))
+- Added the ability to group NFTs in Brave Wallet. ([#8073](https://github.com/brave/brave-ios/issues/8073))
+- Added new design for the Portfolio page. ([#8072](https://github.com/brave/brave-ios/issues/8072))
+- Added method to return "Swap" protocol fees. ([#7924](https://github.com/brave/brave-ios/issues/7924))
+- Added support for Sign-In with Ethereum (EIP-4361). ([#7827](https://github.com/brave/brave-ios/issues/7827))
+- Added support for EIP-6963. ([#7779](https://github.com/brave/brave-ios/issues/7779))
+- Added the ability for users to navigate to ENS domains. ([#6225](https://github.com/brave/brave-ios/issues/6225))
+- Added the ability to select a specific RPC URL as active for Brave Wallet. ([#5902](https://github.com/brave/brave-ios/issues/5902))
+- [Security] Updated Brave Wallet password policy. ([#6093](https://github.com/brave/brave-ios/issues/6093))
+- Implemented "Bridge to Aurora" for Brave Wallet. ([#5785](https://github.com/brave/brave-ios/issues/5785))
+- Updated Portfolio settings page in Brave Wallet. ([#8089](https://github.com/brave/brave-ios/issues/8089))
+- Updated button to create wallet to be disabled when wallet is already being created. ([#8357](https://github.com/brave/brave-ios/issues/8357))
+- Updated “Send” screen in Brave Wallet to not show decimal values for zero balances. ([#8232](https://github.com/brave/brave-ios/issues/8232))
+- Updated “Save Changes” button to be enabled only when there is a change in wallet portfolio settings. ([#8229](https://github.com/brave/brave-ios/issues/8229))
+- Fixed “view my recovery phrase” not displaying in Brave Wallet. ([#8445](https://github.com/brave/brave-ios/issues/8445))
+
+### General 
+
+- Added support for all history data to be synced when "History" sync is enabled. ([#8305](https://github.com/brave/brave-ios/issues/8305))
+- Added setting to ensure YouTube links always load in brave. ([#8272](https://github.com/brave/brave-ios/issues/8272))
+- Added confirmation message for successfully exporting bookmarks. ([#8236](https://github.com/brave/brave-ios/issues/8236))
+- Added confirmation message for resetting configuration in VPN settings. ([#8087](https://github.com/brave/brave-ios/issues/8087))
+- Added support for additional schemas for going to the App Store. ([#8035](https://github.com/brave/brave-ios/issues/8035))
```

Comment:
I could be wrong here but I believe we didn't add any new schema's and just fixed a case where ` itms-apps:` wasn't working in certain cases. So maybe change the above so something as per the following (unless I'm wrong):

```
- Fixed "itms-apps://" links not opening the App Store in certain cases.
```

Reply:
Yeah I was not so sure on that PR. Thanks.